docs/NNNN-direct-clean-storage-layout.md

```diff
@@ -0,0 +1,99 @@
+# OCFL Community Extension NNNN: Direct Clean Storage Layout
+
+* **Extension Name:** NNNN-direct-clean-storage-layout
+* **Authors:** Jürgen Enge, Basel
+* **Minimum OCFL Version:** 1.0
+* **OCFL Community Extensions Version:** 1.0
+* **Obsoletes:** n/a
+* **Obsoleted by:** n/a
+
+## Overview
+
+This object/storage root extension maps content file names or OCFL object identifiers by replacing "dangerous characters" from file and folder names.
```

Comment:
It might be worthwhile to use a few more words here to explicitly talk about how you intend this extension to either be used as a storage layout that maps object ids to storage paths, or as an object extension that maps logical paths to content paths.